src/prefect/tasks/dbt/dbt.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         env: dict = None,
         environment: str = None,
         overwrite_profiles: bool = False,
-        profiles_dir: str = None,
+        profiles_dir: str = os.path.join(os.path.expanduser('~'), '.dbt'),
```

Comment:
This is a little subtle, but if a user builds this task on machine A but runs it on machine B, this directory could be different.  Because of that, I think we should keep this default as `None` and only use this default when none is set (which is what I think your current `run` method implementation handles 👍 )